addons/payment/views/payment_provider_views.xml

```diff
@@ -28,7 +28,8 @@
                 <field name="code" invisible="1"/>
                 <sheet>
                     <!-- === Stat Buttons === -->
-                    <div class="oe_button_box" name="button_box">
+                    <div class="oe_button_box" name="button_box"
+                         attrs="{'invisible': [('module_state', '=', 'uninstalled')]}">
```

Comment:
This is what I was talking about with SEPA:
![image](https://user-images.githubusercontent.com/15896813/212086452-5d1b5490-7ee3-4140-91d0-938d5fa256df.png)


Since the module of SEPA is not `installed`, nor `uninstalled`, but `uninstallable` (because it is in Odoo Enterprise), we need to check `module_state != installed`.
```suggestion
                         attrs="{'invisible': [('module_state', '!=', 'installed')]}">
```

Reply:
@AntoineVDV updated the pr as suggested :heart: